Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +42,36 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            var stop = 0;
+
+            if (array == null)
+            {
+                return;
+            }
+
+            stop = array.Length / 2;
+
+            // Here I take a element and put it into temp. 
+            // So I can put there the element of the other half of the arrau
+            // and put that on that place. After that I can take the contents of
+            // temp and put it on the new place 
+
+            // Example lets say we have this {1,2,3,4,5} 
+            // then temp will be the 1 
```

Comment:
When you explain the algorithm, please don't be too specific. Just go to the main idea.
Now it's too many comments and nevertheless, you forgot to explain about what is the stop variable for.

Reply:
is changed in the new version. Ellmark maked a remarks about it how to make it better

Reply:
It's good enough, just showed how it could be improved. You don't have to change it.